end-to-end-test/shared/wdio/webdriver-manager.conf.js

```diff
@@ -68,10 +74,11 @@ var config = {
         {
             //browserName: 'chrome',
             chromeOptions: {
-                args: [
+                args: _.compact([
```

Comment:
need to do `compact` because when `userDataDir` is undefined webdriver is not able to init Chrome.